AU2/plugins/custom_plugins/CompetencyPlugin.py

```diff
@@ -232,7 +322,12 @@ def on_event_create(self, e: Event, htmlResponse) -> List[HTMLComponent]:
         if self.html_ids["Competency"] in htmlResponse:
             e.pluginState.setdefault(self.identifier, {})[self.plugin_state["COMPETENCY"]] = htmlResponse[self.html_ids["Competency"]]
         if self.html_ids["Attempts"] in htmlResponse:
-            e.pluginState.setdefault(self.identifier, {})[self.plugin_state["ATTEMPTS"]] = htmlResponse[self.html_ids["Attempts"]]
+            # This currently stores attempts as ['a', 'a'] for two attempts by player a

```

Comment:
On this: for storing duplicates of the same id in `attempts` to actually work for granting competency you need to modify competency manager as well; currently it does
```python3
            for assassin_id in e.pluginState.get("CompetencyPlugin", {}).get("attempts", []):
                # Logic for not increasing attempts if player got a player kill in same event.
                for (killer, victim) in e.kills:
                    if killer == assassin_id and not ASSASSINS_DATABASE.get(victim).is_police:
                        break
                else:
                    self.attempts_since_kill[assassin_id] += 1
```
which will only count one attempt per event no matter the number of duplicates!

Reply:
Pretty sure it does work with no changes - it will increment the counter for each attempt and extend competency if ever a multiple of two.
Besides, both the test and manual testing suggests that it works

Reply:
Oh you're right actually
I think I must have misread the first `for` as an `if` or something  😶